docs/testplan/Smartswitch-test-plan.md

```diff
@@ -36,6 +46,14 @@ Purpose of the test is to verify smartswich platform related functionalities/fea
 For every test cases, all DPUs need to be powered on unless specified in any of the case.
 General convention of DPU0, DPU1, DPU2 and DPUX has been followed to represent DPU modules and the number of DPU modules can vary.
 
+## Testbed and Version
+
+The test runs on the os versions 2024011 and above.
+Check is to be in place for that and skips if it is below the mentioned version.
```

Comment:
Could you rephrase with "Add a check to confirm that the test environment uses version 2024011 or later; if the version is earlier, skip the test." or something similar?